src/components/cookie-banner/index.md.njk

```diff
@@ -29,95 +40,112 @@ Check with your organisation's privacy expert to see how data protection legisla
 
 ## How it works
 
-Show the cookie banner when the user accesses your service for the first time.
+Show the cookie banner every time the user accesses your service until they either:
+- accept or reject cookies using the buttons in the cookie banner
+- save their cookie preferences on the service’s [cookie page](/patterns/cookies-page/)
 
-Do not remove the cookie banner or set any non-essential cookies until the user has either:
+Once the user has accepted or rejected cookies, the cookie banner should:
+- hide the cookie banner message
+- show the confirmation message - and a 'hide' button to let the user close the banner
+- set a cookie to save the user’s preferences for 1 year
 
-- accepted or rejected cookies using the buttons in the cookie banner
-- saved their cookie preferences on the service’s [cookie page](/patterns/cookies-page/)
+Make sure the cookie banner does not:
+- show when the user visits again, once their preferences have been saved
+- set any non-essential cookies unless the user accepted them on a previous visit
 
-Position the cookie banner after the opening `<body>` tag and before the ‘skip to main content’ link. If you're using the Nunjucks page template, use the `bodyStart` block.
+Position the cookie banner after the opening `<body>` tag and before the 'skip to main content' link. If you're using the Nunjucks page template, use the `bodyStart` block.
```

Comment:
There's a couple of places in this file where we've switched from smart quotes to straight quotes - is that intentional?

Reply:
```suggestion
Position the cookie banner after the opening `<body>` tag and before the ’skip to main content‘ link. If you're using the Nunjucks page template, use the `bodyStart` block.
```

Reply:
The markdown processor we use supports automatic smart quotes, so it shouldn't matter – they should still render as smart quotes in the Design System.

The exception is the text in the examples, because they don't go through the markdown processor.